lib/infrastructure/models/fx_model.dart

```diff
@@ -1,10 +1,11 @@
+import 'package:app_2i2i/infrastructure/models/user_model.dart';
 import 'package:flutter/material.dart';
 
 @immutable
 class FXModel {
   final int id;
   final DateTime ts;
-  final double value;
+  final double? value;
```

Comment:
when can value be null?

Reply:
This is for null safety, but we don't need, It fixed in the latest commit.